com.spotify.Client.json

```diff
@@ -95,8 +95,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://files.pythonhosted.org/packages/40/9c/107e22f637d33434404c07d69112b9d26b76ee0dd4dd705131ab6cdcc818/python-xlib-0.31.tar.gz",
-                    "sha256": "74d83a081f532bc07f6d7afcd6416ec38403d68f68b9b9dc9e1f28fbf2d799e9"
+                    "url": "https://files.pythonhosted.org/packages/86/f5/8c0653e5bb54e0cbdfe27bf32d41f27bc4e12faa8742778c17f2a71be2c0/python-xlib-0.33.tar.gz",
+                    "sha256": "55af7906a2c75ce6cb280a584776080602444f75815a7aff4d287bb2d7018b32",
```

Comment:
Just had an extra `,` here. You can force push the fix and reopen this.

Reply:
I swear I don't see exactly what is the character causing the issue, I reopened and added your changes. It is still failing, any insight would be greatly appreciated, I thought this would be a simple contribution :(

Reply:
Never mind, I guess I found it, I hate commas so much! :D